client/jetpack-cloud/sections/partner-portal/primary/payment-method-add-v2/index.tsx

```diff
@@ -16,6 +17,12 @@ export default function PaymentMethodListV2() {
 		<Layout className="payment-method-add" title={ title } wide>
 			<LayoutTop>
 				<LayoutHeader>
+					<Breadcrumb
+						items={ [
+							{ label: translate( 'Payment Methods' ), href: '/partner-portal/payment-method' },
```

Comment:
There is one small mistake here

```suggestion
							{ label: translate( 'Payment Methods' ), href: '/partner-portal/payment-methods' },
```

Reply:
good catch. This has been fixed 👍